Mlem/App/Views/Pages/Instance/InstanceDetailsView.swift

```diff
@@ -0,0 +1,296 @@
+//
+//  InstanceStatsView.swift
+//  Mlem
+//
+//  Created by Sjmarf on 20/01/2024.
+//
+
+import MlemMiddleware
+import SwiftUI
+
+struct InstanceDetailsView: View {
+    @Environment(Palette.self) private var palette
+    
+    @State private var showingSlurRegex: Bool = false
+    
+    let instance: any Instance
+    
+    var body: some View {
+        VStack(spacing: 16) {
+            box {
+                HStack {
+                    Label(instance.created.dateString, systemImage: Icons.cakeDay)
+                    Text("•")
+                    Label(instance.created.getRelativeTime(unitsStyle: .abbreviated), systemImage: Icons.time)
+                }
+                .foregroundStyle(.secondary)
+                .font(.footnote)
+            }
+            HStack(spacing: 16) {
+                box {
+                    Text("Users")
+                        .foregroundStyle(.secondary)
+                    Text("\((instance.userCount_ ?? 0).abbreviated)")
+                        .font(.title)
+                        .fontWeight(.semibold)
+                }
+                
+                box {
+                    Text("Communities")
+                        .foregroundStyle(.secondary)
+                    Text("\((instance.communityCount_ ?? 0).abbreviated)")
+                        .font(.title)
+                        .fontWeight(.semibold)
+                        .foregroundStyle(.green)
```

Comment:
This should be part of `palette`--we have several situations where we want an accent color that isn't attached to any particular concept, we can capture them in `palette` as `accent1`, `accent2`, etc.

Reply:
Should these not be entity-attached? `palette.post`, `palette.comment` etc? We use these same colors for the icons in Settings, though we don't use them anywhere else just yet.

Reply:
I suppose we could use numbered accents and then have computed properties of `Palette` for `postAccent`, `commentAccent` etc that return the accent values?

```swift
var postAccent { accent3 }
```

Reply:
That solution sounds perfect to me--we can get very granular semantic colors without needing to redefine forty duplicates in every palette

Reply:
Done 👍